src/adapters/poi/poi.js

```diff
@@ -5,10 +5,9 @@
 export const POI_TYPE = 'poi';
 
 export default class Poi {
-  constructor(id, name, alternativeName, type, latLon, className, subClassName, bbox) {
```

Comment:
Good riddance :+1:

Reply:
yeah, We only get rid of alternativeName for now